### PR TITLE
support both pre- and post-merging data for the "global" baseline

### PR DESCRIPTION
**Pull request by yxdyc**
as the title says

### CODE REVIEW

**Review by rayrayraykk — DISMISSED** [review]
LGTM, but should we provide some baseline yaml files?

File: federatedscope/core/configs/cfg_data.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ def extend_data_cfg(cfg):
     cfg.data.drop_last = False
     cfg.data.sizes = [10, 5]
     cfg.data.shuffle = True
+    cfg.data.server_holds_all = False  # whether the server (client[0]) holds all data, useful in global evaluation case
```
Comment (xieyxclack) [inline federatedscope/core/configs/cfg_data.py:26]: It is confusing to claim server = client[0]

**Review by xieyxclack — COMMENTED** [review]
LGTM

File: federatedscope/core/configs/cfg_data.py (hunk 1 above)
Comment (yxdyc) [reply federatedscope/core/configs/cfg_data.py]: modified accordingly

**Comment by yxdyc** [comment]
> LGTM, but should we provide some baseline yaml files?

added accordingly

**Review by xieyxclack — APPROVED** [review]

**Pull request merged**